target/consensource-api
# Conditionally publish Tarpaulin coverage reports

Signed-off-by: Patrick-Erichsen <Patrick.Erichsen@target.com>

## Proposed change/fix

This PR adds conditional logic to publish Tarpaulin coverage results based on an env var named `SEND_REPORT`. If `SEND_REPORT=1`, the results are published.

## Types of changes

What types of changes does this pull request introduce to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

- `cd test`
- `docker-compose up` 
  - Confirm that the tarpaulin build begins
- `export SEND_REPORT=1`
- `docker-compose up` 
  - Confirm that the build fails 
- `unset SEND_REPORT`
